commentsComponent/schema.sql

```diff
@@ -0,0 +1,20 @@
+DROP DATABASE IF EXISTS commentsList;
+CREATE DATABASE commentsList;
+USE commentsList;
+
+CREATE TABLE users (
+  id INT AUTO_INCREMENT,
+  username VARCHAR (25),
+  PRIMARY KEY (id),    
+);
+
+
+CREATE TABLE comments (
+  id INT AUTO_INCREMENT,
+  parent_id INT,
+  username_id INT,
```

Comment:
I believe you're missing the song_id field in your comments table. If you don't include song_id, I'm not sure you'll have a way to fetch the correct comments for a given song. This should be a foreign key relating to the id of the current song in a songs table. You probably need to implement a songs table in your schema. 

e.g. the user picks a new song. Your client issues a get request for new comments. How is your server going to know which comments to query for? If you pass it a song ID, then you can query for all comments which contain that song_id, and you will be returned the correct comments for the current song.